doc/tutorial/09_retention.md

```diff
@@ -0,0 +1,46 @@
+# Retention Policy
+
+This tutorial will show you how to configure retention to retain backups.
+
+## Preface
+
+This tutorial assumes that you have an installation of PostgreSQL 10+ and pgmoneta.
+
+See [Install pgmoneta](https://github.com/pgmoneta/pgmoneta/blob/main/doc/tutorial/01_install.md)
+for more detail.
+
+## Retention Setup
+
+In `pgmoneta.conf`, you can use `retention = 7, 4, 12, 5` to configure pgmoneta to retain backups
+within the nearest 7 days, 4 weeks, 12 months and 5 years. Specifically, pgmoneta will retain
+all the backups within the nearest 7 days, the latest backup on each Monday within the nearest 4 weeks,
+the latest backup on the first day of each month in the last 12 months and the latest backup on the first
+day of each year in the last 5 years. If you input more than 4 values, pgmoneta will only read the first 4.
+
+
+There are a lot of ways to leave a parameter unspecified. You can omit trailing parameters, 
+for example, you can configure retention as `retention = 7, 4`,
```

Comment:
Remove "for example" - rewrite the sentence. Write what is optional and so on

Reply:
Yeah this part is poorly written. I think the term placeholder will be better.